setup.py

```diff
@@ -32,7 +32,7 @@ def find_package_data(package):
 
 setuptools.setup(
         name='brozzler',
-        version='1.1b11.dev237',
+        version='1.1b11.dev238',
```

Comment:
Incrementing this seems like an anti-pattern. What if two PRs both increment it at the same time? At least splitting to a separate commit would be cleaner?
Maybe this is established practice through; I don't have an alternative to propose without more context.